src/structures/Presence.js

```diff
@@ -35,6 +35,19 @@ class Presence {
      */
     this.activity = activity ? new Activity(this, activity) : null;
 
+    /**
+     * A shard selector, can be
```

Comment:
`shard selector` shouldn't be a thing. in general `null` or `undefined` should be assumed to mean you're dealing with all shards, e.g. a function with the signature `send(data, shard = null) {}`

Reply:
Once again, I enjoy the help from @KingDGrizzle but this was his doing not mine, I'd talk to him about it to see if we should keep it or not. I'm fine either way.